grid_map_pcl/include/grid_map_pcl/PclLoaderParameters.hpp

```diff
@@ -6,92 +6,108 @@
  *      Institute: ETH Zurich, Robotic Systems Lab
  */
 
-#pragma once
+#ifndef GRID_MAP_PCL__PCLLOADERPARAMETERS_HPP_
+#define GRID_MAP_PCL__PCLLOADERPARAMETERS_HPP_
+
+#include <yaml-cpp/yaml.h>
 #include <Eigen/Dense>
+
+#include <rclcpp/rclcpp.hpp>
+
 #include <memory>
 #include <string>
-#include <yaml-cpp/yaml.h>
 
 
-namespace grid_map {
+namespace grid_map
+{
 
 class GridMapPclLoader;
 
-namespace grid_map_pcl {
+namespace grid_map_pcl
+{
 
 class PointcloudProcessor;
 
-class PclLoaderParameters {
-
-	struct DownsamplingParameters {
-		Eigen::Vector3d voxelSize_ { 0.05, 0.05, 0.05 };
-		bool isDownsampleCloud_ = false;
-	};
-
-	struct ClusterExtractionParameters {
-		double clusterTolerance_ = 0.3;
-		unsigned int minNumPoints_ = 2;
-		unsigned int maxNumPoints_ = 1000000;
-	};
-	struct OutlierRemovalParameters {
-		bool isRemoveOutliers_ = false;
-		double meanK_ = 10.0;
-		double stddevThreshold_ = 1.0;
-	};
-	struct RigidBodyTransformation {
-		Eigen::Vector3d translation_ { 0.0, 0.0, 0.0 };
-		Eigen::Vector3d rpyIntrinsic_ { 0.0, 0.0, 0.0 }; // intrinsic rotation (opposite from the ROS convention), order X-Y-Z
-	};
-
-	struct GridMapParameters {
-		double resolution_ = 0.1;
-		unsigned int minCloudPointsPerCell_ = 2;
-	};
-
-	struct Parameters {
-		unsigned int numThreads_ = 4;
-		RigidBodyTransformation cloudTransformation_;
-		OutlierRemovalParameters outlierRemoval_;
-		ClusterExtractionParameters clusterExtraction_;
-		DownsamplingParameters downsampling_;
-		GridMapParameters gridMap_;
-	};
-
-	friend class grid_map::GridMapPclLoader;
-	friend class grid_map::grid_map_pcl::PointcloudProcessor;
+class PclLoaderParameters
+{
+  struct DownsamplingParameters
+  {
+    Eigen::Vector3d voxelSize_ {0.05, 0.05, 0.05};
+    bool isDownsampleCloud_ = false;
+  };
+
+  struct ClusterExtractionParameters
+  {
+    double clusterTolerance_ = 0.3;
+    unsigned int minNumPoints_ = 2;
+    unsigned int maxNumPoints_ = 1000000;
+  };
+  struct OutlierRemovalParameters
+  {
+    bool isRemoveOutliers_ = false;
+    double meanK_ = 10.0;
+    double stddevThreshold_ = 1.0;
+  };
+  struct RigidBodyTransformation
```

Comment:
```suggestion

  struct RigidBodyTransformation
```

Reply:
That should be fixed, not sure why not shown as outdated.